doc/pages/blurbs.json

```diff
@@ -637,5 +637,14 @@
     "link_name": "arun-kumar",
     "link": "https://github.com/Arunthogadiya",
     "social": "github"
+  },
+  {
+    "name": "Ted Vu",
+    "image": "/assets/contributors/ted_vu.jpg",
+    "profile": "I'm a Software Engineer based in Melbourne. I have experienced Mental Health Issue a lot in my life and I want to leverage technologies to address and alleviate the problem. Depression particularly in my country has not been publicly discussed and Vietnamese people tend to have false perceptions about it, they think that having depression means there is something wrong with your mind which causes weird behaviours. In my view, that is not necessarily the case, I believe life is extremely dynamic and dealing with bad time is key to evolve as a human being. Therefore, I really want to contribute to this project as I feel resonated with the project ideas and missions. A nice article in Vietnamese on depression: https://huyenchip.com/vn/depression/",
```

Comment:
I really appreciate this as someone who is Viet as well 💜 

Editing for grammar:

```
I'm a software engineer based in Melbourne. As someone who has experienced mental health Issues throughout my life, I want to leverage technology to address and help alleviate this problem. In my country especially, depression is not as publicly discussed as it should be. Vietnamese people tend to have false perceptions about it. They think that having depression means there is something wrong with your mind and it causes weird behaviours. I believe life is extremely dynamic and dealing with bad times is necessary in evolving as a human. This project resonates with my personal beliefs and mission. Here is <a href="https://huyenchip.com/vn/depression/" target="_blank">helpful article about depression in Vietnam</a>.
```

Reply:
Thanks Julia for reviewing my PRs I'm happy with it, I'm not particularly familiar with Ruby and Rails but I'm pretty sure I can contribute to the projects in many ways. I really like your projects and what you have been working on, please keep up the good work !!